src/UITests/LoadTestFile.cs

```diff
@@ -44,6 +52,120 @@ private void ScanAutomatedChecks()
+        }
+
+        private void ValidateFirstResults()
+        {
+            var resultsList = driver.FindElementByAccessibilityId(AutomationIDs.ScannerResultsDetailsListView);
+            var resultsFailedOnly = resultsList.FindElementsByClassName("ListViewItem");
+            driver.FindElementByAccessibilityId(AutomationIDs.ScannerResultsShowAllButton).Click();
+            var resultsAll = resultsList.FindElementsByClassName("ListViewItem");
+            var fixFollowTb = driver.FindElementByAccessibilityId(AutomationIDs.ScannerResultsFixFollowingTextBox);
+
+            Assert.IsFalse(string.IsNullOrEmpty(fixFollowTb.Text));
+            Assert.AreEqual(2, resultsFailedOnly.Count);
+            Assert.AreEqual(28, resultsAll.Count);
+
+            resultsAll[0].SendKeys(Keys.Control + Keys.Tab);
+        }
+
+        private void ValidateFirstDetails()
+        {
+            var tree = driver.FindElementByAccessibilityId(AutomationIDs.HierarchyControlUIATreeView);
+            var nodes = tree.FindElementsByClassName("TreeViewItem");
+            var props = driver.FindElementByAccessibilityId(AutomationIDs.SnapshotModeControl).FindElementsByClassName("DataGridRow");
+            props[0].Click();
+            var patterns = GetPatternsNodes(AutomationIDs.SnapshotModeControl, nodes);
+
+            Assert.AreEqual("InvokePattern", patterns.First().Text);
+            Assert.AreEqual("Name, Ok", props[0].Text);
+            Assert.AreEqual(3, patterns.Count());
+            Assert.AreEqual(10, props.Count);
+        }
+
+        private void ValidateFirstTree()
+        {
+            var tree = driver.FindElementByAccessibilityId(AutomationIDs.HierarchyControlUIATreeView);
+            var nodes = tree.FindElementsByClassName("TreeViewItem");
+            nodes.First().SendKeys(Keys.Left);
+
+            Assert.AreEqual(16, nodes.Count);
+            Assert.AreEqual("pane 'Desktop 1' has failed test results in descendants.", nodes.First().Text);
+        }
+
+        private void ValidateRootElement()
+        {
+            ValidateRootDetails();
+            ValidateRootResults();
+        }
+
+        private void ValidateRootDetails()
+        {
+            var props = driver.FindElementByAccessibilityId(AutomationIDs.SnapshotModeControl).FindElementsByClassName("DataGridRow");
+            props[0].Click();
+            var text = props[0].Text;
+            var tree = driver.FindElementByAccessibilityId(AutomationIDs.HierarchyControlUIATreeView);
+            var nodes = tree.FindElementsByClassName("TreeViewItem");
+            var patterns = GetPatternsNodes(AutomationIDs.SnapshotModeControl, nodes);
+
+            Assert.AreEqual("Name, Desktop 1", text);
+            Assert.AreEqual(10, patterns.Count());
+            Assert.AreEqual("LegacyIAccessiblePattern", patterns.First().Text);
+            Assert.AreEqual(10, props.Count);
+
+            patterns.Last().SendKeys(Keys.Control + Keys.Tab);
```

Comment:
Having trouble understanding what this does since it's at the end of the method. If it is required for a different test, can we move it into the beginning of the other test?

Reply:
This switches the tab on the right side of ai-win. I could do it at the start of the other test (`ValidateRootResults`), but either way we are making an assumption about the current state of the tabs (currently, we assume the details tab is selected. With the suggested change, we would instead assume the results tab is selected). The benefit to the current way is that we don't have to go and get an element to ctrl + tab on since we already have one here.

Reply:
That makes sense, thanks. Can you add a comment above the line in this or a future PR? I think it might help

Reply:
Comment added!